index.html

```diff
@@ -289,6 +289,20 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+      <div class="contact-form">
+        <div class="description-text">
+          I'm always interested in hearing about new projects, 
+          so if you'd like to chat please get in touch.
+        </div>
+        <div class="form">
+          <form>
+            <input type="text" required maxlength="20" placeholder="Full name">
+            <input type="email" required placeholder="Email address">
+            <textarea maxlength="400" placeholder="Enter text here"></textarea>
```

Comment:
- [ ] As mentioned in the requirement the textarea should  accepts up to 500 chars. So please kinda change the 400 to 500